posts/20240418/index.qmd

```diff
@@ -0,0 +1,38 @@
+---
+title: "Gorilla: Large Language Model Connected with Massive APIs"
+author: "Hongsup Shin"
+date: "2024-04-18"
+image: https://github.com/ShishirPatil/gorilla/raw/gh-pages/assets/img/logo.png
+description: The power of LLMs, in a commercial setting, comes from its ability to use other tools and integrate business domain knowledge too. Unfotunately, it is still challenging getting the LLMs to work well when hooking up custom APIs that interface with custom processes and data. This paper is interesting, largely because it may be a step forward in getting the LLM to accurately use custom tooling.
+categories: [paper]
+---
+
+:::{.callout-note}
+- Paper: [Gorilla: Large Language Model Connected with Massive APIs](https://arxiv.org/abs/2305.15334)
+- Presenter: [Joel](https://www.linkedin.com/in/joelafriyie/)
+- Attendees: [Athula](https://www.linkedin.com/in/athula-pudhiyidath/), [Brian](https://www.linkedin.com/in/brianking387/), [Hongsup](https://www.linkedin.com/in/hongsupshin/)
+:::
+
+## Why this paper?
+
+The power of LLMs, in a commercial setting, comes from its ability to use other tools and integrate business domain knowledge too. Unfotunately, it is still challenging getting the LLMs to work well when hooking up custom APIs that interface with custom processes and data. This paper is interesting, largely because it may be a step forward in getting the LLM to accurately use custom tooling.
+
+## LLM agents
+
+We first discussed what LLM agents are and how they operate. Joel provided us with good introduction. LLM agents are tools with APIs that interact with LLMs to build a customized tool that serves your product's custom needs. [An NVIDIA blog post about LLM agents](https://developer.nvidia.com/blog/introduction-to-llm-agents/) gives an example of using an LLM agent to summarize an earnings call, which requires a complex process more than just looking up an answer. Using an LLM agent is what the paper refers to as "using a tool via API calls."
```

Comment:
*provided us with a good introduction

Reply:
Thanks Joel! I will merge it and publish with the edit :)